lms/templates/student_account/form_field.underscore

```diff
@@ -13,45 +21,47 @@
     <% } %>
 
     <% if ( type === 'select' ) { %>
-        <select id="<%= form %>-<%= name %>"
-            name="<%= name %>"
+        <select id="<%- form %>-<%- name %>"
```

Comment:
These are not spurious -- they were required to pass all the linting issues. (We need to use `-` instead of `=` here).